### PR TITLE
Add a processing to write output directly to a certain folder

### PR DESCRIPTION
**Pull request by jebuss**
* adds a executable to process mcs with an XML, but writes output to a certain output folder
* adds a stream runner where output is not collected via zeroMQ and stored to a central result file.

### CODE REVIEW

File: erna/stream_runner_local_output.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    # if not os.path.isdir(output_directory):
+    #     logger.warn("Output directory {} does not exist. Trying to create it.".format(output_directory))
+    #     try:
+    #         os.mkdir(output_directory, mode=0o755)
+    #     except OSError: # Python >2.5
+    #             pass
+    #     #check if that worked or not
+    #     if not os.access(output_directory, os.W_OK | os.X_OK) :
+    #         logger.error("Cannot write to directory given DEFAULT_TEMP_DIR {} ".format(output_directory))
+    #         return "Cannot write to dir"
```
Comment (MaxNoe) [inline erna/stream_runner_local_output.py]: Why so much outcommented code?

File: erna/stream_runner_local_output.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            logger.error(e)
+            if os.path.exists(output_path):
+                logger.error("Trying to collect output files")
+            else:
+                return "fact-tools error"
+
+        copyfile(output_path, os.path.join(output_dest, output_filename))
+
+        #try to read nans else return empty frame
+        return ft_json_to_df(input_path)
```
Comment (MaxNoe) [inline erna/stream_runner_local_output.py]: Why reread the input dataframe that was written to disk? This should return either nothing or something more useful.

Comment (jebuss) [inline erna/stream_runner_local_output.py]: I will add the output paths of the processed files. This way can see if the full set of runs was processed successfully. We can also think about what to add to the output. right now it's the input path and the output path. So you know which fits input ended in which json lines file.

File: erna/stream_runner_local_output.py (hunk 1 above)
Comment (jebuss) [inline erna/stream_runner_local_output.py]: that was a residual, I will delete it

**Comment by jebuss** [comment]
in general I agree, but on the other hand, I don't want to fill the stream runner with if conditions, which would be necessary, since they write different out put, the stream runners need different input. Don't see a good solution for this right away.

**Pull request merged**